packages/pwa/app/utils/utils.js

```diff
@@ -162,16 +161,55 @@ export const capitalize = (text) => {
 
 /**
  * Get the pwa configuration object from pwa-kit.config.json
- * @param path - path to your object inside pwaKitConfig using jsonPath expression
+ * @param path - path to your target item from pwaKitConfig
  *
- * To read more https://github.com/JSONPath-Plus/JSONPath
+ * @example
+ * getConfig('app.url') => {locale: 'path', site: 'path'}
+ * getConfig('app.sites[0]') => {
+                "id": "RefArch",
+                "alias": "us",
+                "hostnames": [],
+                "l10n": {...}
+            }
+ * @returns {object} - the targeted config value
+ */
+export const getConfig = (path) => {
+    return getObjectProperty(pwaKitConfig, path)
+}
+
+export const isObject = (o) => o?.constructor === Object
+
+/**
+ * Get an object property by a string path separated by dot
  *
- * @example getConfig('app.url') => {locale: 'path', site: 'path'}
- * @returns {object} - the configuration object
+ * @param {object} obj - source object to get data from
+ * @param {string} path - a string path separated each hierarchy by a dot
+ *
+ * @example
+ * const data = {a: 'string 1', b: [{name: 'name 1'}], c: {id: 'abc', child: {id: 1}}}
+ *
+ * getObjectProperty(data, 'a') // return 'string 1'
+ * getObjectProperty(data, 'c.child') // return [{name: 'name 1'}]
+ * getObjectProperty(data, 'b[0].name') // return 'name 1'
+ *
+ * @returns {array|object|string|undefined}
  */
-export const getConfig = (path, opts) => {
-    if (!path) return pwaKitConfig
-    const result = JSONPath({path: `$.${path}`, json: pwaKitConfig, wrap: false, ...opts})
+export const getObjectProperty = (obj, path) => {
```

Comment:
If this is something that you had influence from say a stack overflow post or blog I'd reference it.